src/main/scala/sbtprotoc/ProtocPlugin.scala

```diff
@@ -78,9 +77,18 @@ object ProtocPlugin extends AutoPlugin {
 
       val protocCache = TaskKey[FileCache[ModuleID]]("protoc-cache", "Cache of protoc executables")
 
+      val cacheArtifactResolver = SettingKey[Boolean](
+        "cache-artifact-resolver",
+        "If false, all sandboxed generators will be re-resolved on each invocation. This is useful only when PB.artifactResolver is not referentially transparent."
```

Comment:
Can we make the help text more accessible for end users? Maybe replace the phrase "referentially transparent" with a more concrete description? I think the intent is when `PB.artifactResolver" is expected to return different files on each invocation?

Reply:
52f0b61